langchain4j-elastic/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-parent</artifactId>
+        <version>0.19.0</version>
+        <relativePath>../langchain4j-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>langchain4j-elastic</artifactId>
+    <packaging>jar</packaging>
+
+    <name>LangChain4j integration with Elastic</name>
+    <description>intergration with Elastic to embedding store</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-core</artifactId>
+            <version>0.19.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>co.elastic.clients</groupId>
+            <artifactId>elasticsearch-java</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Can we use gson instead of jackson? We use gson in many places already, would be nice to stick to it if possible (to reduce the number of dependencies)

Reply:
I use jackson because [elastic document](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/8.9/getting-started-java.html) use it. I will try my best to replace jackson with gson.

Reply:
unfortunately, elastic only support for jackson and jsonb. see [Elasticsearch doc](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/8.9/installation.html)

> A JSON object mapping library to allow seamless integration of your application classes with the Elasticsearch API. The Java client has support for [Jackson](https://github.com/FasterXML/jackson) or a [JSON-B](https://github.com/eclipse-ee4j/jsonb-api) library like [Eclipse Yasson](https://github.com/eclipse-ee4j/yasson).